integration/query_test.go

```diff
@@ -39,104 +39,117 @@ func TestQueryBadFindType(t *testing.T) {
 	ctx, collection := s.Ctx, s.Collection
 
 	for name, tc := range map[string]struct {
-		value any
-		err   *mongo.CommandError
+		value      any
+		err        *mongo.CommandError
+		altMessage string
 	}{
 		"Document": {
 			value: bson.D{},
 			err: &mongo.CommandError{
-				Code:    2,
```

Comment:
MongoDB sends different error code for authenticated user. Well, FerretDB will always send the same error regardless or authenticated or not.